src/itest/java/com/iexec/blockchain/IntegrationTests.java

```diff
@@ -38,26 +78,201 @@ public void getMetrics() {
      * TODO when match order is ready
      */
     @Test
-    public void initialize() {
-        initialize("0x06d2fc195ea17e059a176e329ff63bb7f23e5392daded776b9070f127ebd63a8", 0);
+    public void requestInitialize() throws Exception {
+        String appAddress = iexecHubService.createApp(buildRandomName("app"),
+                "docker.io/repo/name:1.0.0",
+                "DOCKER",
+                BytesUtils.EMPTY_HEXASTRING_64,
+                "",
+                30, 1);
+        System.out.println("Created app: " + appAddress);
+        String workerpool = iexecHubService.createWorkerpool(buildRandomName("pool"),
+                30, 1);
+        System.out.println("Created workerpool: " + workerpool);
+        String datasetAddress = iexecHubService.createDataset(buildRandomName("data"),
+                "https://abc.com/def.jpeg",
+                BytesUtils.EMPTY_HEXASTRING_64,
+                30, 1);
+        System.out.println("Created datasetAddress: " + datasetAddress);
+
+        OrderSigner signer = new OrderSigner(chainConfig.getChainId(),
```

Comment:
What about a common `OrderSignerService` to be injected whenever needed. We can keep that for a next PR.

Reply:
Just created the OrderSignerService for you, but IMO we don't need it for now